getRelayConfiguration.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+* @summary Get a Relay configuration
+*/
+
+const { CommunicationIdentityClient } = require("@azure/communication-identity");
+const { CommunicationRelayClient } = require("@azure/communication-network-traversal");
+
+// Load the .env file if it exists
+const dotenv = require("dotenv");
+dotenv.config();
+
+// You will need to set this environment variables or edit the following values
+const connectionString =
+  process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
+
+async function getConfig() {
+  const identityClient = new CommunicationIdentityClient(connectionString);
+  const user = await identityClient.createUser();

```

Comment:
Can you add a comment explaining each step of this `getConfig` function, since this could be used to educate someone as to what this SDK does

Reply:
Especially why we're creating a user identity here

Reply:
Removed the usage of identity in the hero sample, that will be part of the Quickstarts.